tests/integration/tier0/non-destructive/firewalld-inhibitor/main.fmf

```diff
@@ -0,0 +1,22 @@
+summary: |
+    Check that the the c2r inhibits in certain scenario when the firewalld is running
+description: |
+    On Oracle Linux 8.8 there is an issue with the running firewalld process
+    that breaks the conversion. In such case the inhibitor has to be raised.
+
+tier: 0
```

Comment:
```suggestion
tier: 0
```
Also unnecessary, already defined in the [root directory.](https://github.com/oamg/convert2rhel/blob/main/tests/integration/tier0/main.fmf#L8)